crates/vm/levm/src/vm.rs

```diff
@@ -1156,18 +1131,7 @@ impl VM {
 
     /// Gets account, first checking the cache and then the database (caching in the second case)
     pub fn get_account(&mut self, address: Address) -> Account {
-        match cache::get_account(&self.cache, &address) {
-            Some(acc) => acc.clone(),
-            None => {
-                let account_info = self.db.get_account_info(address);
-                let account = Account {
-                    info: account_info,
-                    storage: HashMap::new(),
-                };
-                cache::insert_account(&mut self.cache, address, account.clone());
-                account
-            }
-        }
+        get_account(&mut self.cache, &self.db, address)
```

Comment:
Maybe we could have a different name for the two functions, since having the same one could lead to confusion.

Reply:
hmm maybe, the parameters are different though. They have the same name because they do the same thing but the `VM` one is like a wrapper. A better name doesn't come to my mind, what do you suggest?

Reply:
It's true that there isn't a good name for both, maybe it could be get_account_cache_or_db, but it says more about implementation than anything else. Actual version looks good to me in that sense 👍